include/crm/lrmd.h

```diff
@@ -74,7 +74,6 @@ typedef struct lrmd_key_value_s {
 #define F_LRMD_PROTOCOL_VERSION "lrmd_protocol_version"
 #define F_LRMD_REMOTE_MSG_TYPE  "lrmd_remote_msg_type"
 #define F_LRMD_REMOTE_MSG_ID    "lrmd_remote_msg_id"
-#define F_LRMD_CALLBACK_TOKEN   "lrmd_async_id"
```

Comment:
Just noticed this is public API so I'll have to go back and deprecate it instead

Reply:
I"ll include it in a later PR, where I'm deprecating more LRMD constants.